OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -126,4 +126,17 @@ public interface IPreventsAutoTarget
 	{
 		bool PreventsAutoTarget(Actor self, Actor attacker);
 	}
+
+	[Flags]
+	public enum UncloakType
```

Comment:
When I placed it there, I used `ImpactType` as a reference. I am a bit confused on which should go where. Will put it back to Cloak.cs as initially dont